source/extensions/omni.isaac.lab/docs/CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 ---------
 
+0.26.1 (2024-10-20)
+~~~~~~~~~~~~~~~~~~~
+
+Changed
+^^^^^^^
+
+* Added call to update articulation kinematics after reset to ensure updated states in non-rendering sensors.
```

Comment:
I think it's a good practice to explain what was happening before this change was made. For instance, "Earlier, non-rendering sensors still used older transforms which corrupted the obtained readings."

This way it isn't just a change but also the motivation for the change that gets documented :)